roles/datadog/tasks/pg_stats.yml

```diff
@@ -76,9 +76,16 @@
     RETURNS SETOF pg_catalog.pg_stat_activity AS
     $$ SELECT * from pg_catalog.pg_stat_activity; $$
     LANGUAGE sql VOLATILE SECURITY DEFINER;
-    CREATE VIEW pg_stat_activity_dd AS SELECT * FROM pg_stat_activity();
-    grant SELECT ON pg_stat_activity_dd to datadog;"
+    CREATE VIEW pg_stat_activity_dd AS SELECT * FROM pg_stat_activity();"
   become: yes
   become_user: postgres
   when: stat_views_exist.stdout.find('0 rows') != -1
   notify: restart postgres
+
+- name: grant select to datadog
+  command: |
```

Comment:
Do you think that the following module wold make this task idempotent?

https://docs.ansible.com/ansible/latest/modules/postgresql_privs_module.html

Reply:
Nice @enricostano !

Reply:
We are also looking forward to https://docs.ansible.com/ansible/latest/modules/postgresql_query_module.html#postgresql-query-module which got included in 2.8

Reply:
Me too! :grimacing: